### PR TITLE
🐛 Fixed active state bug in sidebar nav

### LINKED ISSUE #15506: Menu items lose active state when "New ..." modal is open
### Issue Summary

The relevant item in the left-hand menu should remain bold when the "New X" modal is open

image

image

### Steps to Reproduce

1. Open Ghost admin
2. Click on tags, note that "Tags" in the left menu is bold
3. Click on "New tag" and notice that "Tags" in the left menu is no longer bold

Same for Members, Offers, etc.

### Ghost Version

5.16

### Node.js Version

16

### How did you install Ghost?

ghost install

### Database type

MySQL 8

### Browser & OS version

_No response_

### Relevant log / error output

_No response_

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

### PR DESCRIPTION
**Pull request by stephensauceda**
Should close #15506.

The sidebar items that _don't_ open the editor were missing some relative `current-when` attributes so the nested views weren't considered "active".

### CODE REVIEW

**Comment by stephensauceda** [comment]
Added some tests for tags and members. Still in the process of doing so for offers. Noticing the `Offers` menu item isn't present when any of the `acceptance/offers-test.js` tests run. Implies that `settings.paidMembersEnabled : false` in the test environment though it looks like the all the criteria for it being true are present in the fixture. New to Ember so still fumbling about.

File: ghost/admin/tests/helpers/members.js [hunk 1]
```diff
@@ -13,3 +13,9 @@ export function disableMembers(server) {
         ? server.db.settings.update({key: 'members_signup_access'}, {value: 'none'})
         : server.create('setting', {key: 'members_signup_access', value: 'none', group: 'members'});
 }
+
+export function enablePaidMembers(server) {
+    server.db.settings.find({key: 'paid_members_enabled'})
+        ? server.db.settings.update({key: 'paid_members_enabled'}, {value: true})
+        : server.create('setting', {key: 'paid_members_enabled', value: true, group: 'members'});
+}
```
Comment (stephensauceda) [inline ghost/admin/tests/helpers/members.js:21]: I'm not actually sure this is totally the "right" way to go about this. But it's what I found that works, at least. Alternatives I tried:

1.  adding `paid_members_enabled` directly to `mirage/fixtures/settings.js` and then loading that in with `loadFixtures` in the test file. 
2. manually creating the setting with `this.server.create('setting', ...)` in the `beforeEach` of the `beforeEach` of the `as Owner` block of tests.

Neither worked for some reason though I don't ever that could be 💯 user error.

Comment (kevinansfield) [reply ghost/admin/tests/helpers/members.js:21]: This is the right way to do it as it keeps it explicit 👍 The other two approaches are also valid in some cases though so it's interesting they didn't work 🤔

**Review by kevinansfield — APPROVED** [review]

**Comment by ErisDS** [comment]
Hey @stephensauceda thanks so much for this PR 🙏 It's merged now and I've added the `hacktoberfest-approved` label 🎉 . 

This fix will go out in the next release of Ghost (usually Fridays). 

We have many more issues open and more being created all the time, we'd love to see you around again soon 👋

**Pull request merged**